src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from "react";
+
+export interface TabsProps {
+    items: {
+        [key: string]: JSX.Element;
+    };
+    uniqueId: string;
```

Comment:
I think the convention is to use the native attribute names as props, so this should be `id`.